Package.swift

```diff
@@ -36,7 +36,7 @@ let package = Package(
             path: "Sources/Auth"),
         .target(
             name: "WalletConnectRelay",
-            dependencies: ["WalletConnectUtils", "WalletConnectKMS"],
```

Comment:
WalletConnectUtils depends on JSONRPC, do we need to add JSONRPC dependency here also?

Reply:
Removed it